test/interactions/dragInteractionTests.ts

```diff
@@ -1,376 +1,410 @@
-      assert.isFalse(startCallbackCalled, "callback was decoupled from the interaction");
-
-      svg.remove();
-    });
-
-    it("onDrag()", () => {
-      let svg = TestMethods.generateSVG(SVG_WIDTH, SVG_HEIGHT);
-      let c = new Plottable.Component();
-      c.renderTo(svg);
-
-      let drag = new Plottable.Interactions.Drag();
-      let moveCallbackCalled = false;
-      let receivedStart: Plottable.Point;
-      let receivedEnd: Plottable.Point;
-      let moveCallback = (start: Plottable.Point, end: Plottable.Point) => {
-        moveCallbackCalled = true;
-        receivedStart = start;
-        receivedEnd = end;
+      let endPoint = {
+        x: SVG_WIDTH / 2,
+        y: SVG_HEIGHT / 2
       };
-      drag.onDrag(moveCallback);
-      drag.attachTo(c);
-
-      let target = c.background();
-      TestMethods.triggerFakeMouseEvent("mousedown", target, startPoint.x, startPoint.y);
-      TestMethods.triggerFakeMouseEvent("mousemove", target, endPoint.x, endPoint.y);
-      assert.isTrue(moveCallbackCalled, "callback was called on dragging (mousemove)");
-      assert.deepEqual(receivedStart, startPoint, "was passed the correct starting point");
-      assert.deepEqual(receivedEnd, endPoint, "was passed the correct current point");
-
-      receivedStart = null;
-      receivedEnd = null;
-      TestMethods.triggerFakeTouchEvent("touchstart", target, [{x: startPoint.x, y: startPoint.y}]);
-      TestMethods.triggerFakeTouchEvent("touchmove", target, [{x: endPoint.x, y: endPoint.y}]);
-      assert.isTrue(moveCallbackCalled, "callback was called on dragging (touchmove)");
-      assert.deepEqual(receivedStart, startPoint, "was passed the correct starting point");
-      assert.deepEqual(receivedEnd, endPoint, "was passed the correct current point");
-
-      drag.offDrag(moveCallback);
-
-      moveCallbackCalled = false;
-      TestMethods.triggerFakeMouseEvent("mousedown", target, startPoint.x, startPoint.y);
-      TestMethods.triggerFakeMouseEvent("mousemove", target, endPoint.x, endPoint.y);
-      assert.isFalse(moveCallbackCalled, "callback was decoupled from interaction");
-
-      TestMethods.triggerFakeTouchEvent("touchstart", target, [{x: startPoint.x, y: startPoint.y}]);
-      TestMethods.triggerFakeTouchEvent("touchmove", target, [{x: endPoint.x, y: endPoint.y}]);
-      assert.isFalse(moveCallbackCalled, "callback was decoupled from interaction");
-
-      svg.remove();
-    });
-
-    it("onDragEnd()", () => {
-      let svg = TestMethods.generateSVG(SVG_WIDTH, SVG_HEIGHT);
-      let c = new Plottable.Component();
-      c.renderTo(svg);
-
-      let drag = new Plottable.Interactions.Drag();
-      let endCallbackCalled = false;
-      let receivedStart: Plottable.Point;
-      let receivedEnd: Plottable.Point;
-      let endCallback = (start: Plottable.Point, end: Plottable.Point) => {
-        endCallbackCalled = true;
-        receivedStart = start;
-        receivedEnd = end;
+      let outsidePointPos = {
```

Comment:
Probably use "`outsidePointPositive`" to avoid having readers confuse "pos" as short for "position".